fairmq/options/FairMQProgOptions.cxx

```diff
@@ -339,43 +336,23 @@ void FairMQProgOptions::InitOptionDescription()
             ("print-channels",         po::value<bool  >()->implicit_value(true),               "Print registered channel endpoints in a machine-readable format (<channel name>:<min num subchannels>:<max num subchannels>)")
             ("shm-segment-size",       po::value<size_t>()->default_value(2000000000),          "shmem transport: size of the shared memory segment (in bytes).")
             ("shm-segment-name",       po::value<string>()->default_value("fairmq_shmem_main"), "shmem transport: name of the shared memory segment.")
+            ("rate",                   po::value<string>()->default_value(""),                  "rate for conditional run loop (Hz)")
```

Comment:
Can we have the type of this argument directly as a float? Then no need for the stringstream in the code.